guides/common/modules/con_prerequisites-for-foreman-on-aws.adoc

```diff
@@ -0,0 +1,45 @@
+[id="prerequisites-for-{project-context}-on-aws"]
+= Prerequisites for {Project} on AWS
+
+Installing and running {ProjectServer} and {SmartProxyServers} on Amazon Web Services (AWS) has additional requirements to your environment.
+
+* Ensure that the Amazon EC2 instance type meets or exceeds the {InstallingServerDocURL}system-requirements_{project-context}[System requirements] in _{InstallingServerDocTitle}_.
+For the best performance, use an https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/storage-optimized-instances.html[AWS storage optimized instance].
+ifndef::foreman-deb[]
+* Use {InstallingServerDocURL}satellite-storage-requirements_{project-context}[Storage requirements] in _{InstallingServerDocTitle}_ to understand and assign the correct storage to your AWS EBS volumes.
+* Store content on an EBS volume that is separate to the boot volume.
+* Mount your content EBS volume separately in the operating system.
+* Optional: Store other data on a separate EBS volume.
+endif::[]
+* If you want {ProjectServer} and {SmartProxyServer} to communicate using external DNS hostnames, open the required ports for communication in the AWS Security Group that is associated with the instance.
+
+.Amazon Web Service requirements
```

Comment:
It's weird to see this one list of requirements in gerund. Does it mean that I need to do these actions ("Create and access[...]"), that Foreman must be able to do them ("Foreman must be able to create and access[...]"), or that AWS itself needs to be able to do them ("AWS must be able to create[...]")? I know that some of these don't even make sense, but it would be best to clarify by using direct, imperative language that makes it clear who is supposed to do what.